Explorer-Release-Setup-Instructions.md

```diff
@@ -133,8 +133,17 @@ $ docker run -d --restart unless-stopped -p 28967:28967 \
 - Note: If you are using Synology you must add "sudo" in front of commands.
 
 _For Windows Operating Systems use the following line:_
+
 ```bash
-$ docker run -d --restart unless-stopped -p 28967:28967 -e WALLET="0xXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX" -e EMAIL="user@example.com" -e ADDRESS="domain.ddns.net:28967" -e BANDWIDTH="2TB" -e STORAGE="2TB" --mount type=bind,source="<identity-dir>",destination=/app/identity --mount type=bind,source="<storage-dir>",destination=/app/config --name storagenode storjlabs/storagenode:alpha
+$ docker run -d --restart unless-stopped -p 28967:28967 \
```

Comment:
Windows uses different line end escaping. I think it was `^`

Reply:
Ok, I didn't know and I cannot test it